src/script/util/TimeUtil.js

```diff
@@ -21,7 +21,7 @@
 
 import moment from 'moment';
 
-import {t} from 'utils/LocalizerUtil';
+import {t} from './LocalizerUtil';
```

Comment:
Will we change this everywhere?

Reply:
Talked with @atomrc about it. I have a couple of reasons why we should use relative imports. I will sum them up but for now I would like to change only the ones which prevented `tsc` from compiling.

Reply:
We could also use absolute path, that would solve the `..` issue and is also compatible with TS. 
https://hackernoon.com/react-pro-tip-use-absolute-imports-for-better-readability-and-easy-refactoring-2ad5c7f2f957
Let's talk about it altogether